test/integration/stack_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"testing"
+)
+
+func TestStack(t *testing.T) {
+	// Setup
+	client := getClient(t)
+	var err error
+
+	_, err = client.GetStackById(-999)
+	if err == nil {
+		t.Fatal("Expected GetStackById to return an error but got none!")
+		return
+	}
+
+	individualResult, err := client.GetStackByName("THIS_STACK_SHOULD_NOT_EXIST_EVER")
+	if err == nil {
+		t.Fatal("Expected GetStacksByName to NOT return an error but got an err!", err.Error())
```

Comment:
The condition and error message are at odds here. `err == nil` means no error but the message indicates that it did get an error and didn't expect one.

Reply:
updated the copy here, it's basically going to expect an error. i think i was editing this in our pair and lost track mid way as we were moving around